utils/Makefile

```diff
@@ -35,29 +35,31 @@ clean:
 	@rm -fR build dist rpm-build MANIFEST htmlcov .coverage cover ooinstall.egg-info oo-install
 	@rm -fR $(NAME)env
 
+viewcover:
+	xdg-open cover/index.html
+
 virtualenv:
 	@echo "#############################################"
 	@echo "# Creating a virtualenv"
 	@echo "#############################################"
 	virtualenv $(NAME)env
+	. $(NAME)env/bin/activate && pip install setuptools==17.1.1
```

Comment:
Yeah, at some point we'll need to make sure we get fedora-based coverage for this as well.